samples/client/petstore/swift/default/SwaggerClientTests/SwaggerClientTests/PetAPITests.swift

```diff
@@ -74,15 +74,17 @@ class PetAPITests: XCTestCase {
         PetAPI.deletePet(petId: 1000) { (error) in
             // The server gives us no data back so Alamofire parsing fails - at least
             // verify that is the error we get here
-            // Error Domain=com.alamofire.error Code=-6006 "JSON could not be serialized. Input data was nil or zero
-            // length." UserInfo={NSLocalizedFailureReason=JSON could not be serialized. Input data was nil or zero
-            // length.}
-            guard let error = error where error._code == -6006 else {
+            guard let error = error else {
                 XCTFail("error deleting pet")
                 return
             }
-            
-            expectation.fulfill()
+
+            switch error {
+            case ErrorResponse.Error(200, _, _):
```

Comment:
I agree, this is awkard. It's result of the current swift implementation, which doesn't handle `Void` responses currently. Let me see if I can make that go away.